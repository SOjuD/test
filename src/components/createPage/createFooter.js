export const createFooter = (container) => {

	const footer = document.createElement("footer");
	const footerMainBlockConteiner = document.createElement("div");
	const footerMainBlockLogo = document.createElement("div");
	const img = document.createElement("img");

	footer.classList.add("main-block__footer");
	footer.classList.add("footer-main-block");
	footerMainBlockConteiner.classList.add("footer-main-block");
	footerMainBlockConteiner.classList.add("_conteiner");
	footerMainBlockLogo.classList.add("footer-main-block__logo");

	img.src = "./img/icons/logo.svg";
	img.alt = "icon";

	container.append(footer);
	footer.append(footerMainBlockConteiner);
	footerMainBlockConteiner.append(footerMainBlockLogo);
	footerMainBlockLogo.append(img);
}