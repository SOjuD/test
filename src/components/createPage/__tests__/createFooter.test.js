import { screen } from "@testing-library/dom";
import { createFooter } from "../createFooter";

describe("createFooter", () => {
	test("should to be in the document", () => {
		const container = document.createElement("div");

		createFooter(container);

		expect(screen.queryByTestId("footer")).toBeInTheDocument();
	});
});
