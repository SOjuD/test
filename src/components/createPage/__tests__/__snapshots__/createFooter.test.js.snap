// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createFooter should to be in the document 1`] = `null`;
